src/TrailingSlashControllerProvider.php

```diff
@@ -42,12 +42,17 @@ public function connect(Application $app)
             $request = Request::create(
                 '/' . $resource . '/',
                 $app['request']->getMethod(),
-                array_merge($app['request']->query->all(), $app['request']->request->all()),
+                [],
                 $app['request']->cookies->all(),
                 $app['request']->files->all(),
                 $app['request']->server->all(),
                 $app['request']->getContent()
             );
+            $request = $request->duplicate(
```

Comment:
This overwrites the result of `Request::create` from the previous statement. Is this intentional?

Reply:
yes, @biggianteye . `Request::create` doesn't split the values for `query` and `request`, it accepts either body or query string, and processes that according to the method. Former code merged the two values in a single array because of this.

So my case was that I had a post request with query params and a body, and they were being merged together and passed just as the request body. With duplicate, they can be split and handled separately.

Reply:
Oh, never mind. I misread it. I thought you were discarding the previous `$request` value but I see that you are using it as the basis of the new one.